florist/api/launchers/local.py

```diff
@@ -9,10 +9,12 @@
 import flwr as fl
 from fl4health.clients.basic_client import BasicClient
 from fl4health.server.base_server import FlServer
-from flwr.common.logger import DEFAULT_FORMATTER
 from flwr.server import ServerConfig
 
 
+DEFAULT_FORMATTER = logging.Formatter("%(levelname)s %(name)s %(asctime)s | %(filename)s:%(lineno)d | %(message)s")
```

Comment:
I guess flwr removed the DEFAULT_FORMATTER they were exporting previously.

Here there reference it in the documentation so I just sourced it from there: https://flower.ai/docs/framework/how-to-configure-logging.html